packages/forms/resources/views/components/tags-input.blade.php

```diff
@@ -42,7 +42,7 @@
                         x-on:blur="createTag()"
                         x-model="newTag"
                         {{ $getExtraInputAttributeBag()->class([
-                            'block w-full border-0',
+                            'tags-input block w-full border-0',
```

Comment:
Lets stick with utility classes and rename this `webkit-calendar-picker-indicator-opacity-0` maybe?